pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -120,6 +120,10 @@ type TektonHubStatus struct {
 	// The current installer set name
 	// +optional
 	HubInstallerSet map[string]string `json:"hubInstallerSets,omitempty"`
+
+	// The Base64 Encode URI of the Custom Logo
+	// +optional
+	Custom_Logo_Base64_URI string `json:"customLogoUri,omitempty"`
```

Comment:
use camel-case variable name, do not use underscored. `Custom_Logo_Base64_URI` => `CustomLogoBase64URI`
And the input is `data` not `URI`, it can be `CustomLogoBase64Data`

Reply:
I think better you can have it in a struct,
```
CustomLogo {
  Base64Data string
  MediaType string
}
```